### DISCUSSION TITLE
Twitter page 没有自己的最新 Twitter 动态

### DISCUSSION THREAD

**Discussion by guowei-gong** [Ice-Hazymoon/hugo-theme-luna#74] [Q&A] [answered]
嗨，我已经阅读了 discussion 和 issue ，没有发现同样的问题。

## 问题
第一次初始化后，一切正常。Twitter page 中的消息动态与我的 Twitter 动态一致。

随后，我发了一条新的动态，在博客的 Twitter page 中一直没有显示。

我的 Twitter 动态有两条：
![image](https://user-images.githubusercontent.com/62499904/219243185-63bfe795-9693-44f1-ac8c-8d31d627e17e.png)

在 Local 的 Twitter Page 中只有一条动态：
![image](https://user-images.githubusercontent.com/62499904/219243363-36b435d5-dfef-4292-a5dc-a47188b9350a.png)

在 Github Pages 的 Twitter Page 中同样只有一条动态：
![image](https://user-images.githubusercontent.com/62499904/219243649-bebf1acd-945a-433b-aa8c-f2f6c69fbd4b.png)

## 尝试
我尝试在配置中关闭 twitter ，再打开，仍然没有刷新最新的动态。

rsshub 中，我的 [twitter rss](https://rsshub.app/twitter/user/gongguowei01) 有正确的动态数据。

**Comment by Ice-Hazymoon** [comment, marked as answer]
不是实时的，构建一次hugo，更新一次动态，如果是可以用 github action 每日自动构建博客，就能做到每天自动更新

**Reply by guowei-gong** [reply]
感谢你的回复，我会去试试看！你指的构建是我在终端中执行 `hugo`，然后部署吗？因为我本地执行 `hugo` 构建后，在本地通过 `hugo server -D` 预览，没有看到我的最新动态。

**Reply by guowei-gong** [reply]
感谢你提供的漂亮的主题

**Reply by Ice-Hazymoon** [reply]
hugo 对本地开发模式有缓存，应该是12小时，可以在设置中关闭缓存，但是没这个必要，线上自动部署后就会拉去最新的数据

**Reply by guowei-gong** [reply]
使用 github action 后解决了这个问题
